r/tests/testthat/test-compute-aggregate.R

```diff
@@ -383,33 +383,30 @@ test_that("value_counts", {
 })
 
 test_that("any.Array and any.ChunkedArray", {
-  
   data <- c(1:10, NA, NA)
 
-  expect_vector_equal(any(input > 5), data)
+  expect_vector_equal(any(input > 5, na.rm = TRUE), data)
   expect_vector_equal(any(input < 1), data)
   expect_vector_equal(any(input < 1, na.rm = TRUE), data)
   
   data_logical <- c(TRUE, FALSE, TRUE, NA, FALSE)
```

Comment:
To properly test that `any()` is handling `na.rm` consistent with the way base R does, we should test with a vector that does not have any `TRUE` in it. Maybe let's add another few lines of test code here to verify that, and perserve the existing tests too:
```suggestion
  data_logical <- c(TRUE, FALSE, TRUE, NA, FALSE)
  expect_vector_equal(any(input), data_logical)
  expect_vector_equal(any(input, na.rm = FALSE), data_logical)
  expect_vector_equal(any(input, na.rm = TRUE), data_logical)
  
  data_logical <- c(TRUE, FALSE, TRUE, NA, FALSE)
```